CenterForOpenScience/osf.io
# [IN-147][OSF] Show if project is included in a collection

## Purpose

On the project overview page,
Show list of collections that have collected the project.

## Changes

- Add `is_collected`, `linked_collections`, `collected_metadata_queryset` properties to `AbstractNode`
- Add serializer for collection metadata
- Front-end updates to allow hide/show collections details.

Tests for properties are in https://github.com/CenterForOpenScience/osf.io/pull/8249

## QA Notes
Please test these cases:

**Public collection**: shows for any user (logged-in/out)
- No details (`type` and `status`) present: the arrow is not shown
- Some details ( `type` or `status`) present: the arrow shows, and right collection status and type, also make sure upon click the arrows change direction up/down (cross-browsers).

**Private collection**: only shows for contributors on the project

## Documentation

N/A

## Side Effects

N/A

## Ticket

[IN-147](https://openscience.atlassian.net/browse/IN-147)

## After
![included](https://user-images.githubusercontent.com/4511563/38317605-6ee3be40-37fb-11e8-982b-5689198606db.gif)
